webpki-ccadb/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "webpki-ccadb"
+version = { workspace = true }
+edition = { workspace = true }
+readme = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+description = "Common CA Database (CCADB) interface for use with webpki-roots"
+
+[dependencies]
+chrono = { version = "0.4.26", default-features = false, features = ["clock"] }
+csv = "1.2.2"
+hex = { workspace = true }
+num-bigint = "0.4.3"
+pki-types = { workspace = true }
+reqwest = { version = "0.11", features = ["rustls-tls-manual-roots"] }
+rustls-pemfile = "=2.0.0-alpha.2"
+serde = { version = "1.0.183", features = ["derive"] }
+webpki = { workspace = true }
+x509-parser = { workspace = true }
+yasna = { workspace = true }
```

Comment:
Nit: many of your new files lack a trailing newline, would be nice to fix these. GitHub's diff view makes this obvious.